src/goDeclaration.ts

```diff
@@ -102,18 +102,24 @@ function definitionLocation_godef(input: GoDefinitionInput, token: vscode.Cancel
 		return Promise.reject(missingToolMsg + godefTool);
 	}
 	let offset = byteOffsetAt(input.document, input.position);
-	let env = getToolsEnvVars();
 	let p: cp.ChildProcess;
 	if (token) {
 		token.onCancellationRequested(() => killProcess(p));
 	}
 
+	// Set up execFile parameters
+	let env = getToolsEnvVars();
+	env['cwd'] = input.cwd;
+	const goConfig = vscode.workspace.getConfiguration('go', vscode.window.activeTextEditor ? vscode.window.activeTextEditor.document.uri : null);
+	env['timeout'] = getTimeoutConfiguration(goConfig, 'onSave');
```

Comment:
`timeout` and `cwd` is to be set on `options` not `env`. This should be
```
const env = getToolsEnvVars();
const goConfig = ....
const options = {
    env, 
    cwd: input.cwd,
    timeout: getTimeoutConfiguration(goConfig, 'onSave')
}
```

And then pass the `options` to the `cp.execFile` call

Reply:
Same for the other 2 places in this file